com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -35,6 +35,10 @@ internal enum __RpcExecStage
             }
         }
 
+#pragma warning disable IDE1006 // disable naming rule violation check
+        internal virtual string __getBehaviourName() => nameof(NetworkBehaviour);
```

Comment:
so, generally, I put comments on top of ILPP'd things explaining what we'd expect to happen due to ILPP which would help when debugging things at runtime etc.

for instance, there's this comment on top of `__rpc_exec_stage` just a few lines below:
```cs
// RuntimeAccessModifiersILPP will make this `protected`
internal __RpcExecStage __rpc_exec_stage = __RpcExecStage.None;
```

similarly, I'd suggest:
```suggestion
        // NetworkBehaviourILPP will override getter with a method that returns concrete type name string
        internal virtual string __getBehaviourName() => nameof(NetworkBehaviour);
```

Reply:
also one other thing:
should we make this super-obvious by naming it as `__getBehaviourTypeName()` (putting "Type" into the name)?
I worry about people getting confused by behaviour's object name or something else name, maybe verbosity wouldn't harm here?